### ISSUE TITLE
Problems with nested groups

### ISSUE THREAD

**Issue by DaveC49** [MiniKeePass/MiniKeePass#359]
Hi, Love the app. It makes life online a whole lot better and reduces the brain strain. Slight problem is not being able to see nested groups in my main keypass2 file. Is this a limitation of the current version and are there any plans to add this sort of capability?

**Comment by jarush** [comment]
MiniKeePass supports nesting groups within groups.  They should show up on your device like this example screenshot (2 sub-groups inside another group):

![image1](https://cloud.githubusercontent.com/assets/765600/6259891/7a74694e-b79d-11e4-8e13-df1d0177c92f.PNG)

**Comment by DaveC49** [comment]
Thanks Jason,

I realised shortly after making the post that MiniKeePass was not 
picking up the most recent version of the Keypass2 file as Dropbox
was no longer working on my PC. It was uninstalled after an upgrade in 
the operating system went awry and I had to reinstall the operating 
system from scratch and the version of the file on the Dropbox site was 
well and truly outdated.  I now have an up to date version of the file 
which is working correctly with MiniKeePass. my apologies for the confusion.


David
